.github/workflows/tf.validate.yaml

```diff
@@ -37,9 +32,7 @@ jobs:
         fi
 
     - name: Examples
-      run: |
-        terraform -chdir=${{ matrix.module }}/example init
-        terraform -chdir=${{ matrix.module }}/example validate
+      run: ./scripts/examples.sh
```

Comment:
I didn't know u like bash script that much 😆

Reply:
I like reproducibility 🤣